### ISSUE TITLE
doc.json link links to blank page when using http-swagger

### ISSUE THREAD

**Issue by t-evers** [swaggo/swag#389]
When using http-swagger, the generated link "doc.json" points to a blank page instead of the generated swagger.json

To reproduce just generate a swagger documentation as usual with swag init and place code for serving the documentation using http-swagger. The "doc.json"-Link will link to "about:blank"

The link should instead point to the generated swagger.json

I'am using the following versions:
        github.com/swaggo/files v0.0.0-20190110041405-30649e0721f8 // indirect
        github.com/swaggo/http-swagger v0.0.0-20190324132102-654001218d89
        github.com/swaggo/swag v1.5.0

This might be similar to Bug #194, but that one fixes it when using gin to server the documentation.

**Comment by easonlin404** [comment]
fix via https://github.com/swaggo/http-swagger/pull/19

**Comment by easonlin404** [comment]
@t-evers please use the latest http-swagger for it.

**Comment by t-evers** [comment]
@easonlin404 thank you! I fetched the new version an hour ago and tried it but it generates still a link to about:blank.
What I did:
- removed docs folder
- fetched both projects (swag and http-swagger)
`go get -u github.com/swaggo/swag/cmd/swag`
`go get -u github.com/swaggo/http-swagger`
- generated documentation again
`swag init`
- started my service again and opened the documentation. 

All other parts of the served documentation seem to be correct. It's just this "doc.json" link which is leading to a blank page.
